lib/wabbit/connection.ex

```diff
@@ -63,6 +63,8 @@ defmodule Wabbit.Connection do
         Connection.reply(from, :ok)
       {:error, :closed} ->
         :error_logger.format("Connection closed~n", [])
+      {:error, :killed} ->
+        :error_logger.info_msg("Connection closed: shutdown~n", [])
```

Comment:
why `info_msg` instead of `format` like the code above and below uses?

Reply:
because `format/2` is a synonym of `error_msg/2`, which has better "symmetry" with `info_msg/2`. - see http://erlang.org/doc/man/error_logger.html#error_msg-1

Reply:
IOWs, I could make a larger patch changing all calls to `format/2` with `error_msg/2`, and the resulting code would be equivalent, and clearer.

`:erlang.format/2` must be very early/old API...